compiler/luci/pass/src/ResolveCustomOpAddPass.test.cpp

```diff
@@ -17,13 +17,136 @@
 #include "luci/Pass/ResolveCustomOpAddPass.h"
 
 #include <gtest/gtest.h>
+#include <luci/test/TestIOGraph.h>
+
+#include <luci/IR/CircleNodes.h>
+
+using namespace luci::test;
 
 namespace
 {
 
+/**
+ *    graph having Custom operator AddV2
```

Comment:
> graph having Custom operator AddV2

How does this sentence help understand the test?

Reply:
My comment at https://github.com/Samsung/ONE/pull/12509#pullrequestreview-1855170135 suggested to change two empty lines to one but you've removed all two,
which makes readability drop.

Reply:
@seanshpark 
> How does this sentence help understand the test?

Referring to [here](https://github.com/Samsung/ONE/blob/90fe4e6d5b6a8c6246487971fd98cf96df863b0b/compiler/luci/pass/src/ResolveCustomOpBatchMatMulPass.test.cpp#L40), this sentence was written to explain the graph of AddV2 used in the test.
Do you think this setence needs further explanation?


> My comment at https://github.com/Samsung/ONE/pull/12509#pullrequestreview-1855170135 suggested to change two empty lines to one but you've removed all two, which makes readability drop.

Oh.... I will revise it and upload again. :)

Reply:
> explain the graph of AddV2 used in the test.

I don't know how this explains about the test,
There is a AddV2... so what is it have any relation with the test itself?

Reply:
@seanshpark 

well... Thinking about your comment, 
this sentence doesn't seem to be right to explain the test itself. :)

I thought of the following sentence to explain the test according to graph below,
is it okay..? :)

```
/**
<<<<<<<<<<<<<<<<<<<<<<<<<<<
 *   Resolved AddV2 graph in the test looks like below.
 <<<<<<<<<<<<<<<<<<<<<<<<<<<  
 * 
 * [Pattern 1]
 *        [Input]   [BroadcastTo]
 *             \         /
 *           [Custom(AddV2)]
 *                  |
 *             [CustomOut]
 *                  |
 *               [Output]
 * 
 * [Pattern 2]
 *        [Input]   [Custom(BroadcastTo)]
 *             \         /
 *           [Custom(AddV2)]
 *                  |
 *             [CustomOut]
 *                  |
 *               [Output]
 */
```

Reply:
> Resolved AddV2 graph in the test looks like below.

`Test graph with Custom(AddV2) to resolve` ?

Reply:
@seanshpark 
Oh.. I think this sentence is a good idea 
because it's easier to understand the test  :)
I will reflect it and upload soon. Thank you a lot !